data/mods/blazemod/vehicleparts/blob_boards.json

```diff
@@ -0,0 +1,343 @@
+[
```

Comment:
Blob boards painstakingly separated out and consolidated into one file. Unchanged otherwise, could use requirements for a lot of the tiering for gel-ooze-gray but that's a consideration for another PR.